toonz/sources/CMakeLists.txt

```diff
@@ -458,6 +458,8 @@ elseif(BUILD_ENV_UNIXLIKE)
     find_package(LZO REQUIRED)
     message("LZO:" ${LZO_INCLUDE_DIR})
 
+    find_library(PTHREAD_LIBRARY pthread)
```

Comment:
Why is pthread needed?

What is the risk the user does not have pthread installed?

Reply:
This is required it app uses threads, ```#include <pthread.h>``` + any ```pthread_*()``` requires to add this.
On linux and probably mac pthread is somewhere in standard lib, IMHO, that is why this work before. But on BSD pthread requires additional lib for linker.